systemtests/test_env.py

```diff
@@ -29,74 +33,87 @@ class Environment:
     # Whether to perform the bad token test.
     bad_token_test: bool
 
-    def auth_token(self, env_name: str, try_storage: bool) -> str:
-        env_var_name = env_name.upper() + "_AUTH_TOKEN"
+    def env_var_name(self, try_storage: bool) -> str:
+        env_var_name = self.name.upper() + "_AUTH_TOKEN"
         if try_storage:
             env_var_name += "_TRY"
-        return os.environ[env_var_name]
+        return env_var_name
+
+    def help(self) -> str:
+        text = textwrap.dedent(f"""
+            \b
+            {self.name}
+            ====
+              Base URL: {self.base_url}
+              Environment variables:
+                * {self.env_var_name(False)}
+            """)
+        if self.destructive_tests:
+            text += f"    * {self.env_var_name(True)}\n"
+        return text
+
+    def auth_token(self, try_storage: bool) -> str:
+        return os.environ[self.env_var_name(try_storage)]
 
 
 ENVIRONMENTS = {
     "local": Environment(
+        name="local",
         base_url="http://web:8000/",
         destructive_tests=True,
         bad_token_test=True,
     ),
     "stage": Environment(
+        name="stage",
         base_url="https://symbols.stage.mozaws.net/",
         destructive_tests=True,
         bad_token_test=True,
     ),
     "prod": Environment(
+        name="prod",
         base_url="https://symbols.mozilla.org/",
         destructive_tests=False,
         bad_token_test=False,
     ),
     "gcp_stage": Environment(
+        name="gcp_stage",
         base_url="https://tecken-stage.symbols.nonprod.webservices.mozgcp.net/",
         destructive_tests=True,
         bad_token_test=True,
     ),
     "gcp_prod": Environment(
+        name="gcp_prod",
         base_url="https://tecken-prod.symbols.prod.webservices.mozgcp.net/",
         destructive_tests=False,
         bad_token_test=False,
     ),
 }
 
+ENV_HELP = "\n".join(env.help() for env in ENVIRONMENTS.values())
+
+CLI_HELP = f"""
+Runs upload and download tests against the environment specified in ENV_NAME.
+
+Available environments:
+
+{ENV_HELP}
+
+\b
+Usage:
+1. run "make shell" to get a shell in the container
```

Comment:
Note: To run the systemtests against `local`, `make shell` is insufficient. They also need to ensure their local stack is running (i.e. `make run`). Probably worth mentioning?

Reply:
Hmm, I'd think that's clear enough – if you want to test the local environment, you need to run it. How to do that is documented elsewhere, so I'd prefer to restrict this documentation to aspects specific to the system tests. If you forget `make run`, the error message will make that sufficiently clear.